README.md

```
@@ -1,2 +1,72 @@
+## Get commercial support
+
+You can take advantage of a WSO2 on-prem product subscription for the full range of software product benefits needed in your enterprise, like expert support, continuous product updates, vulnerability monitoring, and access to the licensed distribution for commercial use.
+
+To learn more, check [WSO2 Subscription](https://wso2.com/subscription/).
+
+## Can you fill out this survey?
+
+WSO2 wants to learn more about our open source software (OSS) community and your communication preferences to serve you better.
+In addition, we may reach out to a small number of respondents to ask additional questions and offer a small gift.
+
+The survey is available at: [WSO2 Open Source Software Communication Survey
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside link text

(MD039, no-space-in-links)